lib/manager/maven/extract.ts

```diff
@@ -41,6 +41,11 @@ interface MavenProp {
   packageFile: string;
 }
 
+interface MavenPackageFile extends PackageFile {
```

Comment:
```suggestion
export interface MavenPackageFile extends PackageFile {
```
It's used in exported function declaration. Move to `./types.ts` ?